src/screens/Settings/components/OtherSettings/index.tsx

```diff
@@ -280,12 +312,43 @@ export default function OtherSettings({
         </SelectField>
       )}
       {!isWin && (
-        <TextInputField
+        <TableInput
           label={t('options.advanced.title')}
-          htmlId="otherOptions"
-          placeholder={t('options.advanced.placeholder')}
-          value={otherOptions}
-          onChange={handleOtherOptions}
+          htmlId={'enviromentOptions'}
+          header={{
+            key: t('options.advanced.key', 'Key'),
+            value: t('options.advanced.value', 'Value')
+          }}
+          rows={getEnvironmentVariables()}
+          onChange={handleEnviromentVariables}
+          inputPlaceHolder={{
+            key: t('options.advanced.placeHolderKey', 'ENVIORMENT'),
+            value: t(
+              'options.advanced.placeHolderValue',
+              'E.g.: Path/To/ExtraFiles'
+            )
+          }}
+          afterInput={info}
```

Comment:
I think we don't want this `info` block after the inputs anymore, right? it's shown multiple times and doesn't match the new elements in that screen